<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelCreation</class>
 <widget class="QWidget" name="ModelCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Creation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ltr_rc.qrc">
    <normaloff>:/ltr/linuxtrack.svg</normaloff>:/ltr/linuxtrack.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Model Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ModelName"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Model Type</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QRadioButton" name="Model1Pt">
            <property name="text">
             <string>1 Point</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Model3PtCap">
            <property name="text">
             <string>3 Point Cap</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Model3PtClip">
            <property name="text">
             <string>3 Point Clip</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="ModelStack">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="page_3">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="ltr_rc.qrc">:/ltr/cap_small.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_25">
                    <property name="text">
                     <string>A [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QDoubleSpinBox" name="CapA">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>50.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QDoubleSpinBox" name="CapB">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>70.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_26">
                    <property name="text">
                     <string>B [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_27">
                    <property name="text">
                     <string>C [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QDoubleSpinBox" name="CapC">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>92.500000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_28">
                    <property name="text">
                     <string>Hy [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QDoubleSpinBox" name="CapHy">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>100.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_29">
                    <property name="text">
                     <string>Hz [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QDoubleSpinBox" name="CapHz">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>90.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="CapLeds">
                  <property name="text">
                   <string>Model with LEDs</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="ltr_rc.qrc">:/ltr/clip_small.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>A [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QDoubleSpinBox" name="ClipA">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>30.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QDoubleSpinBox" name="ClipB">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>40.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_34">
                    <property name="text">
                     <string>B [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_35">
                    <property name="text">
                     <string>C [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QDoubleSpinBox" name="ClipC">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>70.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="0">
                   <widget class="QLabel" name="label_36">
                    <property name="text">
                     <string>Hy [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="1">
                   <widget class="QDoubleSpinBox" name="ClipHy">
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>-100.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="0">
                   <widget class="QLabel" name="label_37">
                    <property name="text">
                     <string>Hz [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="1">
                   <widget class="QDoubleSpinBox" name="ClipHz">
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>50.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_38">
                    <property name="text">
                     <string>D [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QDoubleSpinBox" name="ClipD">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>80.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_39">
                    <property name="text">
                     <string>Hx [mm]</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QDoubleSpinBox" name="ClipHx">
                    <property name="minimum">
                     <double>-999.990000000000009</double>
                    </property>
                    <property name="maximum">
                     <double>999.990000000000009</double>
                    </property>
                    <property name="value">
                     <double>-100.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="ClipLeds">
                  <property name="text">
                   <string>Model with LEDs</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QCheckBox" name="SinglePtLeds">
                    <property name="text">
                     <string>Model with LEDs</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="ModelDescStack">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page_5">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>Here you can enter sizes of the CAP model.
All values are in milimeters.

Uncheck the &quot;Model with LEDs&quot; checkbox 
when using reflective model like TrackClip 
or similar. </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_6">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>Here you can enter sizes of the CLIP model.
All values are in milimeters.

Uncheck the &quot;Model with LEDs&quot; checkbox 
when using reflective model like TrackClip 
or similar.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>301</width>
           <height>501</height>
          </rect>
         </property>
         <property name="text">
          <string>Here you can select whether you single point
is made of LED (or another light source),
or reflective material.
</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CreateButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ltr_rc.qrc"/>
 </resources>
 <connections/>
</ui>
